boards/radxa/default.nix

```
@@ -0,0 +1,16 @@
+{ rockchipRK399 }:
+
+{
+  radxa-RockPi4C = rockchipRK399 {
+    boardIdentifier = "radxa-RockPi4C";
```

```suggestion
    boardIdentifier = "radxa-rockPi4C";
```

The board name is `camelCased` and not `PascalCased`. Otherwise good noticing it is `Cased` :)